src/Paket.Core/Common/Utils.fs

```diff
@@ -560,7 +555,7 @@ let downloadFromUrl (auth:Auth option, url : string) (filePath: string) =
             do! task
         with
         | exn ->
-            failwithf "Could not download from %s%s Message: %s%s" url Environment.NewLine exn.Message (innerText exn)
+            raise <| Exception(sprintf "Could not download from '%s'" url, exn)
```

Comment:
We almost never should do `failwithf` in a general `with exn ->` clause, not even when we embed the message (like here)...
I really think F# should make this more easy (with some syntax), even the F# compiler is doing it wrong everywhere

/cc @dsyme any ideas?
maybe allowing to use comma in combination with `failwithf "some message", innerException`